File: Java/topflytechCodec/src/com/topflytech/codec/entities/InnerGeoDataMessage.java
package com.topflytech.codec.entities;

import java.util.ArrayList;
import java.util.Date;

public class InnerGeoDataMessage extends Message{
    private Date date;
    private ArrayList<InnerGeofence> geoList = new ArrayList<>();
    public int getLockGeofenceEnable() {
        return lockGeofenceEnable;
    }

    public void setLockGeofenceEnable(int lockGeofenceEnable) {
        this.lockGeofenceEnable = lockGeofenceEnable;
    }

    private int lockGeofenceEnable;
    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public ArrayList<InnerGeofence> getGeoList() {
        return geoList;
    }

    public void setGeoList(ArrayList<InnerGeofence> geoList) {
        this.geoList = geoList;
    }

    public void addGeoPoint(InnerGeofence geo){
        this.geoList.add(geo);
    }
}

File: TopflytechCodec/C#/TopflytechCodec/TopflytechCodec/Entities/InnerGeoDataMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class InnerGeoDataMessage : Message
    {
        private DateTime date;
        private List<InnerGeofence> geoList = new List<InnerGeofence>();
        private int lockGeofenceEnable;

        public int LockGeofenceEnable
        {
            get { return lockGeofenceEnable; }
            set { lockGeofenceEnable = value; }
        }

        public DateTime Date
        {
            get { return date; }
            set { date = value; }
        }

        public List<InnerGeofence> GeoList
        {
            get { return geoList; }
            set { geoList = value; }
        }

        public void AddGeoPoint(InnerGeofence geo)
        {
            this.geoList.Add(geo);
        }
    }
}
